src/components/dashboard/Dashboard.js

```diff
@@ -272,9 +272,12 @@ const Dashboard = props => {
   //currently it seems too complicated to make it its own effect as it both depends on "feeds" and changes them
   //which would lead to many unwanted subscribe/unsubscribe
   const subscribeToFeed = async () => {
```

Comment:
just do not call for this fn, getFeedPage() or userStorage.getAllFeed() until initializedRegistered = true

Reply:
you'd need to add
const { initializedRegistered = false } = userStorage

then add initializedRegistered to the effect initializing everything and run initialization when initializedRegistered === true

Reply:
@johnsmith-gooddollar i'm not sure this is the correct way in this case.
you do not want to block rendering of dashboard.
and reading from feed is called from the feed list, so in this case adding `await feedstorage` might be the best option.
but i'm not sure it is required when subscribing... probably it is required in getFeedPage